akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -135,9 +193,21 @@ final case class `If-Range`(entityTagOrDateTime: Either[EntityTag, DateTime]) ex
 }
 
 // FIXME: resurrect SSL-Session-Info header once akka.io.SslTlsSupport supports it
-final case class RawHeader(name: String, value: String) extends jm.headers.RawHeader {
+final class RawHeader(val name: String, val value: String) extends jm.headers.RawHeader {
```

Comment:
Do we *have* to drop the `case class` here?
It seems we are loosing the `toString` implementation and `copy` method as well, no?